obsplus/waveforms/get_waveforms.py

```diff
@@ -56,14 +53,15 @@ def get_waveforms(
     Stream
     """
     stream = stream.copy()
+    time1, time2 = UTC(starttime or SMALL_UTC), UTC(endtime or BIG_UTC)
     st = stream.select(
         network=network, station=station, location=location, channel=channel
     )
-    st = st.trim(starttime=UTC(starttime or SMALL_UTC), endtime=UTC(endtime or BIG_UTC))
+    st.trim(starttime=time1, endtime=time2)
     return st
 
 
-def get_waveforms_bulk(st: Stream, bulk: List[str], **kwargs) -> Stream:
+def get_waveforms_bulk(st: Stream, bulk: bulk_waveform_arg_type, **kwargs) -> Stream:
```

Comment:
Looks like the doc-string might be out of date here - index is made from the st?